internal/utils_test.go

```diff
@@ -233,6 +233,123 @@ data:
 	}
 }
 
+func TestGetPolicyTemplateFromPolicyTypeManifest(t *testing.T) {
+	t.Parallel()
+	tmpDir := t.TempDir()
+	manifestFiles := []types.Manifest{}
+
+	// Test manifest is non-root IAM policy type
+	IamManifestPath := path.Join(tmpDir, "configmapIam.yaml")
+	IamManifestYAML := `
+apiVersion: policy.open-cluster-management.io/v1
+kind: IamPolicy
+metadata:
+  name: policy-limitclusteradmin-example
+spec:
+  severity: medium
+  namespaceSelector:
+    include: ["*"]
+    exclude: ["kube-*", "openshift-*"]
+  remediationAction: enforce
+  maxClusterRoleBindingUsers: 5
+`
+
+	err := ioutil.WriteFile(IamManifestPath, []byte(IamManifestYAML), 0o666)
+	if err != nil {
+		t.Fatalf("Failed to write %s", IamManifestPath)
+	}
+
+	manifestFiles = append(
+		manifestFiles, types.Manifest{Path: IamManifestPath},
+	)
+
+	policysetManifestPath := path.Join(tmpDir, "configmapPolicyset.yaml")
+	policysetManifestYAML := `
+apiVersion: policy.open-cluster-management.io/v1beta1
+kind: PolicySet
+metadata:
+  labels:
+    open-cluster-management.io/policy-set: openshift-hardening
+  name: openshift-hardening
+  namespace: policies
+spec:
+  description: ""
+  policies:
+  - policy-limitclusteradmin
+`
+
+	err = ioutil.WriteFile(policysetManifestPath, []byte(policysetManifestYAML), 0o666)
+	if err != nil {
+		t.Fatalf("Failed to write %s", policysetManifestPath)
+	}
+
+	manifestFiles = append(
+		manifestFiles, types.Manifest{Path: policysetManifestPath},
+	)
+
+	// Write a bogus file to ensure it is not picked up when creating the policy
+	// template
+	bogusFilePath := path.Join(tmpDir, "README.md")
```

Comment:
This is already tested else where, so could you please remove this to make the tests "slimmer"?